app/services/rag_wise/rag_workflow_service.rb

```diff
@@ -22,8 +22,11 @@ def initialize(course, evaluation_service, character)
   end
 
   def get_assistant_response(post, topic)
-    query_payload = "query title: #{topic.title} query text: #{post.text} "
+    sanitised_query = ActionController::Base.helpers.strip_tags(post.text)
+    query_payload = "query title: #{topic.title} query text: #{sanitised_query} "
```

Comment:
Do you not need to sanitize the `topic.title` here as well?

Reply:
Actually, if I am not wrong topic title by default does not have any html tags. But I can just sanitise as well